dotCMS/src/integration-test/java/com/dotcms/datagen/FolderDataGen.java

```diff
@@ -21,7 +21,10 @@
     private String fileMasks = "";
     private Folder parent;
     private Host site = host;
-
+    private String defaultFileType = CacheLocator.getContentTypeCache()
```

Comment:
Codacy found an issue: [Field defaultFileType has the same name as a method](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6325390)